Apps/frontend/src/views/SettingsPopup.svelte

```diff
@@ -0,0 +1,127 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  import {
+    getChannelConfig,
+    postChannelConfig,
+  } from "../rest/ChannelConfigController";
+  import {
+    getLiveChannelConfig,
+    setLiveChannelConfig,
+    availableChannelConfigPresets,
+    presetPopupOpen,
+  } from "../stores";
+  import {
+    LABEL_HEADER_LOAD_FROM_LIST_CHANNEL_CONFIG,
+    LABEL_HEADER_CREATE_CHANNEL_CONFIG,
+  } from "../labels";
+  import { EVENT_LOADED_CHANNEL_CONFIG } from "../events";
+  import { getAllChannelConfig } from "../rest/ChannelConfigController";
+
+  let presetName;
+  let selectedPreset;
+
+  const dispatch = createEventDispatcher();
+
+  async function loadChannelConfigById() {
+    var response = await getChannelConfig(selectedPreset);
+
+    if (response !== undefined) setLiveChannelConfig(response.channels);
+
+    dispatch(EVENT_LOADED_CHANNEL_CONFIG, {});
+  }
+
+  async function storeChannelConfig() {
+    await postChannelConfig(getLiveChannelConfig(), presetName);
+    
+    var response = await getAllChannelConfig();
+    if (response !== undefined) availableChannelConfigPresets.set(response);
+    
+  }
+
+  function handleSaveButton() {
+    if (document.getElementById("presetName").value === "") {
+      document.getElementById("storeChannelConfig").disabled = true;
+    } else {
+      document.getElementById("storeChannelConfig").disabled = false;
+    }
+  }
+
+  function handleLoadButton() {
+    if (document.getElementById("availablePreset").value === "") {
```

Comment:
Do we have to do it that way? Or can we e.g. use `bind:this` on the selectand then use sth. like `availablePresetSelect.value`? (or here simply use the bound value `selectedPreset`?)

Same idea for other `document.getElementById`

Reply:
Did some small optimizations